src/extension.ts

```diff
@@ -598,6 +598,34 @@ function launchMetals(
       });
     });
 
+    // TODO replace with ServerCommand when done
+    registerTextEditorCommand(
+      `metals.copy-worksheet-output`,
```

Comment:
- [ ] Replace the hardcoded value with `CopyWorksheetOutput` after `metals-languageclient` is release.